src/main/resources/db/migration/V0006__Create_table_animal_images.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `animal_images` (
```

Comment:
Why we divided sql queries to more one file ?

Reply:
I guess that for migration and develop DB must be more than  one file.Step by stap. If I am wrong, I will make one file.

Reply:
First: You don't have real server. 
Second: This can be considered the first migration. These files can be combined into one file. In the following tasks you can create new migrations

Reply:
done